sys/vane/jael.hoon

```diff
@@ -1604,7 +1712,10 @@
     %-  put-request
     :+  /filter/new  `'new filter'
     :*  %eth-new-filter
-        `[%number ?:((lte latest-block 40) 0 (sub latest-block 40))]
+        `number+latest-block
+        ::  XX We want to load from a snapshot at least 40 blocks behind, then
+        ::  replay to the present
+        ::  `[%number ?:((lte latest-block 40) 0 (sub.add latest-block 40))]
```

Comment:
40 still feels really arbitrary, and longer reorgs are not entirely out of the question. Of course, nothing a re-`%init` can't solve...